assets/charts/components/metrics-server/Chart.yaml

```diff
@@ -12,4 +12,4 @@ maintainers:
 name: metrics-server
 sources:
 - https://github.com/kubernetes-incubator/metrics-server
-version: 2.11.1
+version: 2.11.2
```

Comment:
This chart still ships metrics-server v0.3.6, we could upgrade to v0.3.7.

Reply:
why not wait for the official release of the helm chart ?

Reply:
I'm fine with waiting, I'm just saying there is new image available :)